index.js

```diff
@@ -1,6 +1,30 @@
 /* eslint-env node */
 'use strict';
 
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var MergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
   name: 'ember-i18next',
+
+  included() {
+    this._super.included.apply(this, arguments);
+    this.import('vendor/i18next.js');
+    this.import('vendor/shims/i18next.js');
+    this.import('vendor/i18nextXHRBackend.js');
+    this.import('vendor/shims/i18next-xhr-backend.js');
+  },
+
```

Comment:
The code below finds i18next dependencies in the addon's `node_modules` tree and adds them to the consuming app's vendor tree, along with the shims.